web/src/settings_components.ts

```diff
@@ -1528,5 +1528,13 @@ export function create_group_setting_widget({
         }
     }
 
+    const $group_setting_label = $pill_container.siblings(".group-setting-label");
+    $group_setting_label.on("click", () => {
+        // Click opens the typeahead.
+        $pill_container.find(".input").expectOne().trigger("click");
+        // Focus puts the cursor into the input.
+        $pill_container.find(".input").expectOne().trigger("focus");
```

Comment:
This opens the typeahead even when user is not allowed to change the setting.

Reply:
[Fixed](https://github.com/zulip/zulip/compare/4cbe6a2e6c244026bd3c7d5d83043a812e3e37ba..5018be3a0770d60ddf81cec3863722fab4e7ab5c)